package.json

```diff
@@ -40,6 +40,7 @@
     "react-redux": "^8.1.2",
     "resize-observer-polyfill": "^1.5.1",
     "tailwindcss": "3.3.2",
+    "tailwindcss-scrollbar": "^0.1.0",
```

Comment:
Can you send a link to the NPM package used here?

Reply:
https://www.npmjs.com/package/@gradin/tailwindcss-scrollbar